tools/azsdk-cli/Azure.Sdk.Tools.Cli/README.md

```diff
@@ -29,69 +49,82 @@ The Azure SDK Engineering System's SDK developer experience MCP server that is i
 
 ```text
 Description:
-  azsdk cli - A Model Context Protocol (MCP) server that enables various tasks for the Azure SDK Engineering 
-  System.
+  azsdk cli - A Model Context Protocol (MCP) server that enables various tasks for the Azure SDK Engineering System.
 
 Usage:
   azsdk [command] [options]
 
 Options:
-  --tools <tools>        If provided, the tools server will only respond to CLI or MCP server requests for      
-                         tools named the same as provided in this option. Glob matching is honored.
+  --tools <tools>        If provided, the tools server will only respond to CLI or MCP server requests for tools named the same as provided in this option. Glob matching is honored.
   --debug                Enable debug logging [default: False]
   -o, --output <output>  The format of the output. Supported formats are: plain, json [default: plain]
   --version              Show version information
   -?, -h, --help         Show help and usage information
 
 Commands:
-  azp                  Azure Pipelines Tool
-  eng                  Internal azsdk engineering system commands
-  log                  Log processing commands
-  start                Starts the MCP server (stdio mode)
-  release-plan         Manage release plans in AzureDevops
-  spec-tool            TypeSpec project tools for Azure REST API Specs
-  spec-pr              Pull request tools
-  spec-workflow        Tools to help with the TypeSpec SDK generation.
-  validate-typespec    Run typespec validation
-  releaseReadiness     Checks release readiness of a SDK package.
-  hello-world <input>  Tests echoing a message back to the client
+  eng                       Internal azsdk engineering system commands
+  download-prompts
+  log                       Log processing commands
+  validate-workspace-files
+  start                     Starts the MCP server (stdio mode)
```

Comment:
These commands that are MCP-lifecycle related should be under an 'mcp' top level command. I know you're just documenting what is already in place, but it's not the only purpose for this tool and having all the MCP commands at the top level can be confusing.